src/intersection-observer-polyfill.js

```diff
@@ -294,16 +291,12 @@ export class IntersectionObserverPolyfill {
     const ampdoc = Services.ampdoc(element);
     if (ampdoc.win.MutationObserver && !this.mutationObserver_) {
       this.viewport_ = Services.viewportForDoc(element);
-      this.mutationPass_ = new Pass(ampdoc.win, () => {
-        if (this.viewport_) {
-          this.tick(this.viewport_.getRect());
-        }
-      });
       this.mutationObserver_ = new ampdoc.win.MutationObserver(
           this.handleMutationObserverNotification_.bind(this)
       );
       this.mutationObserver_.observe(ampdoc.getRootNode(), {
         attributes: true,
+        attributeList: ['hidden'],
```

Comment:
Hoo, I didn't know about this. Actually, looking at [MDN](https://developer.mozilla.org/en-US/docs/Web/API/MutationObserverInit), it should be [`attributeFilter`](https://developer.mozilla.org/en-US/docs/Web/API/MutationObserverInit/attributeFilter).

Reply:
Oops! Wrote that line at the DMV will fix now